clients/linux/src/backend.rs

```diff
@@ -30,20 +30,20 @@ macro_rules! match_core_err {
     (
         $err:expr,
         $enum:ident,
-        $( $variants:ident => $matches:expr ),+,
+        $( $variants:ident $( ( $val:ident ) )? => $matches:expr ),*,
```

Comment:
Why the change from `+` to `*`?

Reply:
I was experimenting and forgot to change it back. My bad.

Reply:
Ah ok :+1: